install_pi_zero_host.sh

```
@@ -0,0 +1,611 @@
+if [ -f package.json ]; then
+  STACK="node"
+elif [ -f requirements.txt ] || [ -f pyproject.toml ]; then
+  STACK="python"
+fi
+
+echo "Detected stack: $STACK"
+
+# -----------------------------
+# Node stack install/build (frontend + backend)
+# -----------------------------
+if [ "$STACK" = "node" ]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)